README.md

```diff
@@ -137,27 +132,20 @@ Install-packageProvider -Name NuGet -Force
 Install-Module -Name AzureRM -Force
 ```
 
- 1. Run Microsoft Powershell.
- 1. Create the local parameters file by calling
-   ```
-   Invoke-Webrequest -Uri "https://raw.githubusercontent.com/teradici/deploy/master/azuredeploy.parameters.json" -OutFile "my.azuredeploy.parameters.json"
-   ```
- 3. Modify <samp>my.azuredeploy.parameters.json</samp> to include the necessary deployment parameters.
- 1. Run the following script, substituting username, password, resource group name, and desired region:
-
-```
-$spUsername = "<username>@<example>.com"
-$spPass = ConvertTo-SecureString "<password>" -AsPlainText -Force
-$cred = New-Object -TypeName pscredential -ArgumentList $spUsername, $spPass
-Login-AzureRMAccount -Credential $cred
-
-$azureRGName = "<rgname>"
-New-AzureRMResourceGroup -Name $azureRGName -Location "East US"
-New-AzureRMResourceGroupDeployment -DeploymentName "ad1" -ResourceGroupName $azureRGName -TemplateFile "https://raw.githubusercontent.com/teradici/deploy/master/azuredeploy.json" -TemplateParameterFile "my.azuredeploy.parameters.json"
-
-```
-
-If you do not want credentials in the file just go directly to <samp>Login-AzureAccount</samp> without the <samp>-Credential</samp> parameter and it will give you a prompt.
+1. Run Microsoft Powershell.
+1. Log into Azure using one of the following methods
+
+    a. Log in using Service Principal Account credentials:
+    ```
+    $spTenantId = "<Service-Principal-Tenant-ID>"
+    $cred = Get-Credentials
+    Login-AzureRMAccount -Credential $cred -tenantId $spTenantId
+    ```
+    b. Log in using Azure Credentials
+    ```
+    Login-AzureRMAccount
```

Comment:
I think Microsoft is pushing people towards Add-AzureRMAccount as the preferred alias for this method.

Reply:
They're latest documentation still seems to use Login https://docs.microsoft.com/en-us/powershell/azure/authenticate-azureps?view=azurermps-5.0.0

Reply:
Fair enough. The IDE was whining at me so I switched over. Your call.